wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -52,18 +52,28 @@ <h2>{% if postelection.is_london_assembly_additional %}Additional members{% else
               You will have one vote, and can vote for a single party list or independent candidate.
             {% else %}
               {% if postelection.winner_count and postelection.election.voting_system_id == 'FPTP' %}
-                You will have up to {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}.
+                You will have {{ postelection.winner_count|intword }} vote{{ postelection.winner_count|pluralize }}, 
+                and can choose from {{ postelection.people.count }} candidate{{ postelection.people|pluralize }}. 
+              {% endif %}
+              {% if postelection.winner_count and postelection.election.voting_system_id == 'AMS' %}
+                You will have one vote, and can choose from <strong>{{ postelection.party_ballot_count }}</strong>
+                in the {{ postelection.friendly_name }}.
+              {% endif %}
+              {% if postelection.winner_count and postelection.election.voting_system_id == 'sv' %}
+                You will have two votes, and can choose from <strong>{{ postelection.party_ballot_count }}</strong>
+                in the {{ postelection.friendly_name }}.
+              {% endif %}
+              {% if postelection.winner_count and postelection.election.voting_system_id == 'STV' %}
+                You can choose from {{ postelection.people.count }} candidate{{ postelection.people|pluralize }}. 
```

Comment:
Same for all these `postelection.people.count` - can we pipe them to `apnumber`, so all `1`s become `one` etc